# Description of configuration files can be found at "https://gitlab.ethz.ch/ics-group/projects/andrea/crs-2.0/-/wikis/ros-configuration"
# node params
node_rate: 30 # hz
show_track_angle: false # If yes, publishes arrow instead of line of track center. This slows down visualization
point_downsampling_factor: 10 # only render every ith point of the track

trajectory:
  show_past_est_trajectory: false
  show_past_gt_trajectory: false
  number_of_past_samples: 300

